site/content/en/docs/start/_index.md

```diff
@@ -534,7 +534,7 @@ minikube dashboard
 
 <h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">4</strong></span>Deploy applications</h2>
 
-Create a sample deployment and expose it on port 8080:
+Create a sample deployment and expose it on port 8080, if you are using apple m1, replace the image with `k8s.gcr.io/echoserver-arm:1.8`:
```

Comment:
Suggestion: please capitalize proper nouns -`Apple M1` - for reference.

Reply:
ok

Reply:
> 

Done

Reply:
It's a good suggestion. The modification has been completed